### Eclipse Workspace Patch 1.0
#P Drupal_CVS
Index: nodefamily/nodefamily.module
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/nodefamily/nodefamily.module,v
retrieving revision 1.19.2.14
diff -u -r1.19.2.14 nodefamily.module
--- nodefamily/nodefamily.module	4 Oct 2007 08:53:48 -0000	1.19.2.14
+++ nodefamily/nodefamily.module	12 Nov 2007 16:41:57 -0000
@@ -31,6 +31,15 @@
   return $items;
 }
 
+/**
+ * Additional permission  to allow users to create profiles for other users
+ */
+function nodefamily_perm() {
+    return array(
+        'create others nodes',
+	);
+}
+
 
 /**
  * Menu callback; provide the means to add and remove relations between
@@ -183,7 +192,7 @@
         break;
         
     case 'validate':
-        if (!$node->nid && user_access('administer nodes')) {
+        if (!$node->nid && ( user_access('administer nodes') || user_access('create others nodes')) ) {
           if ($account = user_load(array('name' => $node->name))) {
             $node->uid = $account->uid;
           }
@@ -198,7 +207,14 @@
         }
         break;
     case 'submit':
-        if ($node->nid && user_access('administer nodes')) {
+        // adjust uid if $node->nid && user_access('create others nodes'):
+        // since node_submit() is not aware of 'create others nodes'!!
+        if (!$node->nid && user_access('create others nodes')) {
+          if ($account = user_load(array('name' => $node->name))) {
+            $node->uid = $account->uid;
+          }
+        }
+        if ($node->nid && ( user_access('administer nodes') || user_access('create others nodes') ) ) {
           $parents = nodefamily_get_parent_types($node->type);
           $children = nodefamily_get_child_types($node->type);
           $oldnode = node_load($node->nid);          
@@ -256,7 +272,7 @@
   }
   else if (($form['#node_type']) && $form['#node_type']->type .'_node_form' == $form_id) {
     $node = $form['#node'];
-    if (!$node->nid && !user_access('administer nodes') && nodefamily_content_type_is_max($node->type, $node->uid)) {
+    if (!$node->nid && !user_access('administer nodes') && !user_access('create others nodes') && nodefamily_content_type_is_max($node->type, $node->uid)) {
       drupal_set_message(t("You can't create any more nodes of this type."), 'error');
       $form = NULL;
     }
