? mw.patch
? tmp.patch
Index: views_cache.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/views/Attic/views_cache.inc,v
retrieving revision 1.2.2.18
diff -u -F^f -r1.2.2.18 views_cache.inc
--- views_cache.inc	14 Jul 2007 19:12:02 -0000	1.2.2.18
+++ views_cache.inc	28 Sep 2007 20:57:11 -0000
@@ -276,14 +276,9 @@ function _views_get_query(&$view, $args,
   else {
     views_load_query();
     $info = _views_build_query($view, $args, $filters);
-    $data = array(
-        'query' => _views_replace_args($info['query'], $info['args']),
-        'countquery' => _views_replace_args($info['countquery'], $info['args']),
-    );
     if ($view->is_cacheable) {
-      cache_set('views_query:' . $view->name, 'cache_views', serialize($data));
+      cache_set('views_query:' . $view->name, 'cache_views', serialize($info));
     }
-    $info = array_merge($info, $data);
   }
 
   // Run-time replacement so we can do cacheing
@@ -332,4 +327,4 @@ function _views_get_style_plugins($title
     }
   }
   return ($titles ? $views_style_plugins['title'] : $views_style_plugins['base']);
-}
\ No newline at end of file
+}
Index: views_query.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/views/Attic/views_query.inc,v
retrieving revision 1.51.2.11
diff -u -F^f -r1.51.2.11 views_query.inc
--- views_query.inc	12 Apr 2007 15:18:59 -0000	1.51.2.11
+++ views_query.inc	28 Sep 2007 20:57:12 -0000
@@ -659,16 +659,3 @@ function _views_build_summary(&$query, $
     return $query;
   }
 }
-
-/*
- * Support db_query() style argument replacement in a clause.
- */
-
-function _views_replace_args($clause, $args) {
-  // cut & pasted directly from db_query.
-  if (isset($args[0]) and is_array($args[0])) { // 'All arguments in one array' syntax
-    $args = $args[0];
-  }
-  _db_query_callback($args, TRUE);
-  return preg_replace_callback(DB_QUERY_REGEXP, '_db_query_callback', $clause);
-}
