Index: views_query.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/views/Attic/views_query.inc,v
retrieving revision 1.51.2.11
diff -u -r1.51.2.11 views_query.inc
--- views_query.inc	12 Apr 2007 15:18:59 -0000	1.51.2.11
+++ views_query.inc	29 Sep 2007 17:19:18 -0000
@@ -139,7 +139,7 @@
   if (is_array($filter['value']) && count($filter['value'])) {
     if ($filter['operator'] == 'OR' || $filter['operator'] == 'NOR') {
       $query->ensure_table($table);
-      $where_args = array_merge(array($query->use_alias_prefix . $table, $field), $filter['value']);
+      $where_args = array_merge(array($query->get_table_name($table), $field), $filter['value']);
       $placeholder = array_fill(0, count($filter['value']), '%s');
       if ($filter['operator'] == 'OR') {
         $query->add_where("%s.%s IN ('". implode("','", $placeholder) ."')", $where_args);
@@ -167,7 +167,7 @@
   }
   else {
     $query->ensure_table("$table");
-    $query->add_where("%s.%s %s '%s'", $query->use_alias_prefix . $table, $field, $filter['operator'], $filter['value']);
+    $query->add_where("%s.%s %s '%s'", $query->get_table_name($table), $field, $filter['operator'], $filter['value']);
   }
 }
 
@@ -554,7 +554,7 @@
    * @param $alias_prefix
    *   An optional prefix for the table alias.
    */
-  function get_table_name($table, $table_num, $alias_prefix = null) {
+  function get_table_name($table, $table_num = 1, $alias_prefix = null) {
     if (is_null($alias_prefix)) {
       $alias_prefix = $this->use_alias_prefix;
     }
Index: views.module
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/views/views.module,v
retrieving revision 1.166.2.43
diff -u -r1.166.2.43 views.module
--- views.module	14 Jul 2007 18:54:16 -0000	1.166.2.43
+++ views.module	29 Sep 2007 17:19:17 -0000
@@ -1928,7 +1928,7 @@
     case 'handler':
       $table = $filterinfo['table'];
       $column = $filterinfo['field'];
-      $field = "$table.$column";
+      $field = $query->get_table_name($table) .'.'. $column;
       $query->ensure_table($table);
 
       switch ($filter['operator']) {
@@ -2020,7 +2020,7 @@
 
   $table = $filterinfo['table'];
   $column = $filterinfo['field'];
-  $field = "$table.$column";
+  $field = $query->get_table_name($table) .'.'. $column;
   if ($filterinfo['from_unixtime']) {
     $field = "from_UNIXTIME($field)";
   }
Index: modules/views_user.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/views/modules/Attic/views_user.inc,v
retrieving revision 1.14.2.7
diff -u -r1.14.2.7 views_user.inc
--- modules/views_user.inc	3 Mar 2007 22:31:27 -0000	1.14.2.7
+++ modules/views_user.inc	29 Sep 2007 17:19:18 -0000
@@ -372,7 +372,7 @@
  */
 function views_handler_filter_uid_touched($op, $filter, $filterinfo, &$query) {
   if (!module_exists('comment')) { // not much to do in this case
-    $query->add_where("node.uid = '%s'", $filter['value']);
+    $query->add_where($query->get_table_name('node') .".uid = '%s'", $filter['value']);
     return;
   }
 
@@ -383,7 +383,7 @@
   $table = $query->get_table_name('comments', $tblnum);
 
   $query->set_distinct(); // once you do this, there's just one node.
-  $query->add_where("node.uid = '%s' OR $table.uid = '%s'", $filter['value'], $filter['value']);
+  $query->add_where($query->get_table_name('node') .".uid = '%s' OR $table.uid = '%s'", $filter['value'], $filter['value']);
 }
 
 /*
@@ -412,18 +412,18 @@
       if (in_array(DRUPAL_ANONYMOUS_RID, $filter['value'])) {
         return;
       }
-      $query->add_where("node.uid != 0");
+      $query->add_where($query->get_table_name('node') .".uid != 0");
       return;
     }
     else if ($filter['operator'] == 'NOR') {
       //filter for anonymous users too?
       if (in_array(DRUPAL_ANONYMOUS_RID, $filter['value'])) {
         $query->ensure_table($table);
-        $query->add_where("%s.%s = '0'", $table, $field); //this will get no results
+        $query->add_where("%s.%s = '0'", $query->get_table_name($table), $field); //this will get no results
         return;
       }
       else {
-        $query->add_where("node.uid = 0");
+        $query->add_where($query->get_table_name('node') .".uid = 0");
         return;
       }
     }
@@ -433,7 +433,7 @@
       //if there are other roles left the default filter handling is ok
       //else we must exclude the anonymous user
       if (count($filter['value']) == 0) {
-        $query->add_where("node.uid != 0");
+        $query->add_where($query->get_table_name('node') .".uid != 0");
         return;
       }
     }
@@ -445,7 +445,7 @@
       unset($filter['value'][$key]);
       
       if (count($filter['value']) == 0) {
-        $query->add_where("node.uid = 0");
+        $query->add_where($query->get_table_name('node') .".uid = 0");
         return;
       }
       else {
@@ -456,15 +456,15 @@
         $num = $query->add_table($table, false, 1, $joininfo);
         $tablename = $query->get_table_name($table, $num);
         //and allow anonymous nodes
-        $query->add_where("node.uid = 0 OR %s.%s IS NOT NULL", $tablename, $field);
+        $query->add_where($query->get_table_name('node') .".uid = 0 OR %s.%s IS NOT NULL", $tablename, $field);
         return;
       }
     }
     else if ($filter['operator'] == 'NOR') {
-      $query->add_where("node.uid != 0");
+      $query->add_where($query->get_table_name('node') .".uid != 0");
     }
     else if ($filter['operator'] == 'AND') {
-      $query->add_where("node.uid = 0");
+      $query->add_where($query->get_table_name('node') .".uid = 0");
       if (count($filter['value']) == 1) {
         //no need to run the default filter handler any more
         return;
Index: modules/views_upload.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/views/modules/Attic/views_upload.inc,v
retrieving revision 1.4.4.5
diff -u -r1.4.4.5 views_upload.inc
--- modules/views_upload.inc	3 Mar 2007 22:40:31 -0000	1.4.4.5
+++ modules/views_upload.inc	29 Sep 2007 17:19:18 -0000
@@ -160,12 +160,13 @@
 function views_handler_file_filter_fid_exist($op, $filter, $filterdata, &$query) {
   switch ($op) {
     case 'handler':
-      $query->ensure_table('file_revisions');
+      $table = 'file_revisions';
+      $query->ensure_table($table);
       if ($filter['value']) {
-        $query->add_where('file_revisions.fid');
+        $query->add_where($query->get_table_name($table) .'.fid');
       }
       else {
-        $query->add_where('ISNULL(file_revisions.fid)');
+        $query->add_where('ISNULL('. $query->get_table_name($table) .'.fid)');
       }
   }
 }
