--- trunk/drupal/5.x/themes/garland_cichorieae/page.tpl.php	(revision 11237)
+++ trunk/drupal/5.x/themes/garland_cichorieae/page.tpl.php	(revision 15595)
@@ -74,14 +74,5 @@
 		<div id="menu">
           <?php if (isset($primary_links)) : ?>
-
-            <?php
-              foreach($primary_links as &$link) {
-                $link_title = $link['title'];
-                if(module_exists('cdm_api')){
-                  $link_title = generalizeString($link_title);
-                }
-                $link['attributes']['target'] = $link_title;
-              }
-              print theme('links', $primary_links, array('class' => 'links primary-links')) ?>
+            <?php print theme('links', $primary_links, array('class' => 'links primary-links')) ?>
           <?php endif; ?>
           <?php if (isset($secondary_links)) : ?>
Index: trunk/drupal/5.x/themes/garland_cichorieae/template.php
===================================================================
