--- trunk/drupal/5.x/themes/garland_cichorieae/template.php	(revision 15127)
+++ trunk/drupal/5.x/themes/garland_cichorieae/template.php	(revision 15798)
@@ -471,5 +471,5 @@
 
  	$mediaQueryParameters = array("type"=>"ImageFile");
-	$galleryLinkUri = path_to_taxon($taxon->uuid).'/images';
+	$galleryLinkUri = path_to_taxon($taxon->uuid, 'images');
 
     // cdm_dataportal_show_media = ???? TODO
Index: trunk/drupal/5.x/themes/diptera/diptera.profile
===================================================================
